src/Progress/CircularProgress.js

```diff
@@ -98,7 +112,7 @@ function CircularProgress(props) {
         },
         className,
       )}
-      style={{ width: size, height: size, ...style }}
+      style={{ width: size, height: size, ...rootStyle, ...style }}
```

Comment:
Should we move `width: size, height: size` to `rootStyle`?

Reply:
They're prop based.